### PR TITLE
Task5

### PR DESCRIPTION
**Pull request by 233gme**
Added styled-components and new styles into App component

### CODE REVIEW

File: src/Components/Card/index.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  if (props.onView) {
+    return defaultMode;
+  } else {
+    return !editModeFlag ? defaultMode : editMode;
+  }
+  
+};
+
+export default Card;
```
Comment (iba-dhetsikau) [inline src/Components/Card/index.jsx:101]: Please be more attentive. I've described you a bug in the previous PR, it was not resolved.

> Actually you have a bug. Here is the scenario:
> 
> 1. Start editing the card.
> 2. Check ViewOnly checkbox.
> 3. Uncheck ViewOnly checkbox.
> 4. The card appears in editMode, however, should be in normalMode.

File: src/Components/Card/index.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    setCardState(!editModeFlag);
+  };
+
+  // Adding new text to cards
+  const [textData, setNewTextData] = useState({
+    ...props.card
+  });
+  const addNewText = event => {
+    textData[event.target.name] = event.target.value;
+    setNewTextData(textData);
```
Comment (iba-dhetsikau) [inline src/Components/Card/index.jsx]: Looks a bit odd to me. You're first directly mutating the state, then re-assign it with the same values? This is bad practice. You can just call setNewTextData with object copy by using spread operator:
```
const addNewText = event => {
  setNewTextData({
    ...textData,
    [event.target.name] = event.target.value,
    });
}
```

**Review by iba-dhetsikau — CHANGES_REQUESTED** [review]
Please apply changes according to file comments and resend for a review.

File: src/Components/Card/index.jsx (hunk 1 above)
Comment (233gme) [reply src/Components/Card/index.jsx:101]: Fixed

File: src/Components/Card/index.jsx (hunk 2 above)
Comment (233gme) [reply src/Components/Card/index.jsx]: Yes I agree...
Fixed

**Review by iba-dhetsikau — APPROVED** [review]
Merge approved.

**Pull request merged**